src/data/storage/StorageDataStoreClient.ts

```diff
@@ -72,4 +72,49 @@ export class StorageDataStoreClient extends StorageClient {
     public async listKeys(): Promise<string[]> {
         return this.dataStore.getKeys().getData();
     }
+
+    public async getObjectSharing(key: string): Promise<ObjectSharing | undefined> {
+        const metadata = await this.getMetadataByKey(key);
+
+        const sharing = _.omit(
+            metadata,
+            "created",
+            "lastUpdated",
+            "lastUpdatedBy",
+            "namespace",
+            "key",
+            "value",
+            "favorite",
+            "id"
+        );
```

Comment:
Not 100% in with the ``omit`` approach, it _should_ be safe to use if we are certain about the response of the API endpoint but it's normally safer to do the negate ``pick`` logic.

In the constant storage I've implemented it this way:

```ts
    public async getObjectSharing(key: string): Promise<ObjectSharing | undefined> {
        const {
            user,
            userAccesses,
            userGroupAccesses,
            publicAccess,
            externalAccess,
        } = await this.getConstant(key);

        return { user, userAccesses, userGroupAccesses, publicAccess, externalAccess };
    }
```

Reply:
Ok, fixed!